stable/app/templates/deployment.yaml

```
@@ -1,3 +1,4 @@
+{{- if not .Values.container.enabled }}
```

the logic is incorrect
```suggestion
{{- if .Values.container.enabled }}
```